Actions/VerifyPRChanges/VerifyPRChanges.ps1

```diff
@@ -0,0 +1,34 @@
+Param(
+  [Parameter(HelpMessage = "The token to use for the GitHub API", Mandatory = $false)]
+  [string] $token,
+  [Parameter(HelpMessage = "Base commit of the PR", Mandatory = $false)]
+  [string] $baseSHA,
+  [Parameter(HelpMessage = "Head commit of the PR", Mandatory = $false)]
+  [string] $headSHA,
+  [Parameter(HelpMessage = "The name of the repository the PR is going to", Mandatory = $false)]
+  [string] $prBaseRepository,
+  [Parameter(HelpMessage = "The URL of the GitHub API", Mandatory = $false)]
+  [string] $githubApiUrl
```

Comment:
So, this parameter is here because it's needed in the in PS and there `github.api_url` is not in the context, right?

I don't particularly like the approach of having parameters to sake of matching YAML to PS. Also, with a bit more complex actions (that's not just calling a PS script), this whole idea collapses. Up to you to decide what's better, but in my opinion, it's not worth polluting the actions so that we can test that they call PS with the same parameters.

Reply:
I suppose in this case it's such a simple parameter, that we could just put it in the script instead of passing it as a parameter

I agree on your point about the Yaml tests though